src/main/java/dk/easv/dal/dao/CustomerDAO.java

```diff
@@ -0,0 +1,52 @@
+package dk.easv.dal.dao;
+
+import dk.easv.be.Customer;
+import dk.easv.be.User;
+import dk.easv.dal.ConnectionManager;
+import dk.easv.dal.interafaces.ICRUDDao;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.concurrent.ConcurrentMap;
+
+public class CustomerDAO implements ICRUDDao<Customer> {
+    private PreparedStatement preparedStatement;
+    private ConnectionManager connectionManager;
+
+    @Override
+    public int add(Customer object) throws SQLException {
+        if (object == null) {
+            String sql = "INSERT INTO customer (name ,email ,address ,zipcode) VALUES (?,?,?,?)";
```

Comment:
add inserted.id change spacing between ",".  Use this as reference: Insert Into contents (image, text) OUTPUT inserted.id VALUES (?, ?);